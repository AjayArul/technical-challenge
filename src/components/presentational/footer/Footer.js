import React from 'react'
import {FOOTER_TEXT} from './../../../Strings'

const Footer = () => {
    return (
        <footer className="container-fluid text-center border-top pt-3">
            <p>{FOOTER_TEXT}</p>
        </footer>   
    )
}

export default Footer