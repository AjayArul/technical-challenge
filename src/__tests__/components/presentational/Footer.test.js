import React from 'react'
import Footer from './../../../components/presentational/footer/Footer';

describe('Footer test <Footer />', () => {
  let _wrapper
  beforeEach(() => {
    _wrapper = shallow(<Footer />)
  })
  it('Should Defined and render <Footer /> component', () => {
      expect(_wrapper).toBeDefined()
      expect(_wrapper).toMatchSnapshot()
  });
});
