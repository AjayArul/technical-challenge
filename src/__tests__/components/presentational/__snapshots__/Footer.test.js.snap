// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer test <Footer /> Should Defined and render <Footer /> component 1`] = `ShallowWrapper {}`;
